homework/index.js

```diff
@@ -30,13 +30,93 @@
     return elem;
   }
 
+  function renderRepository(leftContainer, repository) {
+    const tabel = createAndAppend('tabel', leftContainer, { id: 'tabel' });
+    const tr1 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr1, { class: 'col1_data', text: 'Repository:' });
+    createAndAppend('td', tr1, { class: 'col2_data', text: repository.name });
+
+    const tr2 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr2, { class: 'col1_data', text: 'Description:' });
+    createAndAppend('td', tr2, { class: 'col2_data', text: repository.description });
+
+    const tr3 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr3, { class: 'col1_data', text: 'Forks:' });
+    createAndAppend('td', tr3, { class: 'col2_data', text: repository.forks });
+
+    const tr4 = createAndAppend('tr', tabel, { class: 'descriptions' });
+
+    createAndAppend('td', tr4, { class: 'col1_data', text: 'Updated:' });
+    createAndAppend('td', tr4, { class: 'col2_data', text: repository.updated_at });
+  }
+
+  function renderContributors(rightContainer, url) {
+    fetchJSON(url, (err, contributors) => {
+      if (err !== null) {
+        createAndAppend('div', rightContainer, { text: err.message, class: 'alert-error' });
+      } else {
+        createAndAppend('p', rightContainer, { class: 'contributions', text: 'Contributions' });
+        const ul = createAndAppend('ul', rightContainer, { class: 'listOfcontributors' });
+        contributors.forEach(contributor => {
+          const li = createAndAppend('li', ul);
+          const subdiv = createAndAppend('div', li, { class: 'subdiv' });
+          createAndAppend('img', subdiv, { src: contributor.avatar_url, class: 'image' });
+          createAndAppend('a', subdiv, {
+            class: 'contUrl',
+            text: contributor.login,
+            href: contributor.html_url,
+            target: '_blank',
+          });
+          createAndAppend('p', subdiv, {
+            class: 'noOfContributions',
+            text: contributor.contributions,
+          });
+        });
+      }
+    });
+  }
+
   function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
+    const root = document.getElementById('root');
+    fetchJSON(url, (err, repositories) => {
+      if (err !== null) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const header = createAndAppend('header', root, { id: 'header', text: 'HYF Repositories' });
+
+        const select = createAndAppend('select', header, { id: 'repo_options' });
+        repositories.sort((a, b) => {
```

Comment:
The main function also looks good, but the sorting function here could also be extracted into a separate function in order to simplify the code.